app/helpers/application_mailer_helper.rb

```diff
@@ -0,0 +1,12 @@
+module ApplicationMailerHelper
+  def email_footer
+    uuid = Setting[:instance_id]
+    title = Setting[:instance_title]
+
+    if uuid.present? && title.present?
+      _('This email was sent from Foreman instance %{title} identified by %{uuid}') % {title: title, uuid: uuid}
+    elsif uuid.present?
+      _('This email was sent from Foreman instance with UUID %{uuid}') % { uuid: uuid }
```

Comment:
I'd keep it consistent with the prior line:
```suggestion
      _('This email was sent from Foreman identified by %{uuid}') % { uuid: uuid }
```